plugins/mysql/mysql_disk_by_user

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python3
+# -*- python -*-
+# -*- coding: utf-8 -*-
+#
+# Plugin to monitor MySQL database disk usage per user
+#
+# Usage: Place in /etc/munin/plugins/ (or link it there using ln -s)
+#   Add this to your /etc/munin/plugin-conf.d/munin-node:
+#       [mysql_disk_size]
+#       user mysql
+#       users_with_underscores foo_bar d_ritchie # usernames of users in the
+#                                                # system that include an underscore
+#       limit 1024000 # minimum db size in order to report a specific user,
+#                     # in bytes (defaults to 5M). "no" for none.
+#
+# 2019-07-25: v 1.0 pcy <pcy.ulyssis.org>: created
+#
+# Parameters understood:
+#
+#   config   (required)
+#   autoconf (optional - used by munin-config)
+#
+# magic markers:
+#%# family=auto
+#%# capabilities=autoconf
+
+import os
+import re
+import subprocess as sp
+import sys
+
+
+def weakbool(x):
+    u = x.upper().strip()
+    if u == "FALSE" or u == "NO" or u == "N":
+        return False
+    try:
+        return bool(int(x))
+    except ValueError:
+        return bool(x)
+
+
+LIMIT = os.getenv('limit', str(50000 * 1024))
```

Comment:
I think, you are using 50M as a default here (instead of 5M advertised in the documentation).

Reply:
Oops, you're right, fixed.